nise/template-manifest.json

```diff
@@ -0,0 +1,187 @@
+    }, {
+        "category": "lineItem",
+        "name": "UnblendedRate"
+    }, {
+        "category": "lineItem",
+        "name": "UnblendedCost"
+    }, {
+        "category": "lineItem",
+        "name": "BlendedRate"
+    }, {
+        "category": "lineItem",
+        "name": "BlendedCost"
+    }, {
+        "category": "lineItem",
+        "name": "LineItemDescription"
+    }, {
+        "category": "lineItem",
+        "name": "TaxType"
+    }, {
+        "category": "lineItem",
+        "name": "LegalEntity"
+    }, {
+        "category": "product",
+        "name": "ProductName"
+    }, {
+        "category": "product",
+        "name": "group"
+    }, {
+        "category": "product",
+        "name": "groupDescription"
+    }, {
+        "category": "product",
+        "name": "location"
+    }, {
+        "category": "product",
+        "name": "locationType"
+    }, {
+        "category": "product",
+        "name": "operation"
+    }, {
+        "category": "product",
+        "name": "productFamily"
+    }, {
+        "category": "product",
+        "name": "region"
+    }, {
+        "category": "product",
+        "name": "servicecode"
+    }, {
+        "category": "product",
+        "name": "servicename"
+    }, {
+        "category": "product",
+        "name": "sku"
+    }, {
+        "category": "product",
+        "name": "usagetype"
+    }, {
+        "category": "pricing",
+        "name": "publicOnDemandCost"
+    }, {
+        "category": "pricing",
+        "name": "publicOnDemandRate"
+    }, {
+        "category": "reservation",
+        "name": "AmortizedUpfrontCostForUsage"
+    }, {
+        "category": "reservation",
+        "name": "AmortizedUpfrontFeeForBillingPeriod"
+    }, {
+        "category": "reservation",
+        "name": "EffectiveCost"
+    }, {
+        "category": "reservation",
+        "name": "EndTime"
+    }, {
+        "category": "reservation",
+        "name": "ModificationStatus"
+    }, {
+        "category": "reservation",
+        "name": "NormalizedUnitsPerReservation"
+    }, {
+        "category": "reservation",
+        "name": "RecurringFeeForUsage"
+    }, {
+        "category": "reservation",
+        "name": "StartTime"
+    }, {
+        "category": "reservation",
+        "name": "TotalReservedNormalizedUnits"
+    }, {
+        "category": "reservation",
+        "name": "TotalReservedUnits"
+    }, {
+        "category": "reservation",
+        "name": "UnitsPerReservation"
+    }, {
+        "category": "reservation",
+        "name": "UnusedAmortizedUpfrontFeeForBillingPeriod"
+    }, {
+        "category": "reservation",
+        "name": "UnusedNormalizedUnitQuantity"
+    }, {
+        "category": "reservation",
+        "name": "UnusedQuantity"
+    }, {
+        "category": "reservation",
+        "name": "UnusedRecurringFee"
+    }, {
+        "category": "reservation",
+        "name": "UpfrontValue"
+    }],
+    "charset": "UTF-8",
+    "compression": "GZIP",
```

Comment:
We're not specifically using the manifest to get the file compression right now. Instead we're making a call to `DescribeReportDefinitions`. It might still be nice to make compression a variable here and match it with the report file being uploaded.

Reply:
now a parameter in the template. Defaulted to 'GZIP' in code as I'm now compressing the CSV file.